data_steward/cdr_cleaner/cleaning_rules/remove_aian_participants.py

```diff
@@ -0,0 +1,71 @@
+"""
+Background
+DC-685
+There is a requirement from the NIH that participants who identified as AI/AN in PPI cannot have ANY of their data
+included in released datasets for the Data Browser or Workbench/Cohort Builder. These participants have had their data
+retracted non-programmatically previously. However, since this is expected to be a long-term policy, this should be a
+cleaning rule applied to combined so this can be automated.
+We will remove ALL DATA in combined associated with PIDs who have rows with the following in the observation table:
+observation_source_concept_id = 1586140
+value_source_concept_id = 1586141
+"""
+
+# Project imports
+from cdr_cleaner.cleaning_rules import sandbox_and_remove_pids
+from notebooks import bq
+
+TICKET_NUMBER = 'DC685'
+
+PIDS_QUERY = """
+SELECT person_id
+FROM `{project}.{dataset}.observation`
+WHERE observation_source_concept_id IN(1586140) AND value_source_concept_id IN(1586141)
```

Comment:
optional:  Consider adding a space between `IN(`.  Just makes it more readable should we need to debug later.